.eslintrc.json

```diff
@@ -38,6 +38,7 @@
         "@typescript-eslint/no-use-before-define": 0,
         "class-methods-use-this": 0,
         "no-useless-constructor": 0,
+        "import/no-cycle": 0,
```

Comment:
There are legit reasons why dependencies may cycle through the `utils` file, suppressing this as a result. Happy to discuss